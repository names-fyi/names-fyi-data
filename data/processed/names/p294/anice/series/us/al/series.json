[{"y":1920,"t":5,"m":0,"f":5,"mr":null,"fr":530,"mp":0,"fp":0.0001626439398867998},{"y":1924,"t":5,"m":0,"f":5,"mr":null,"fr":540,"mp":0,"fp":0.0001536853752996865}]