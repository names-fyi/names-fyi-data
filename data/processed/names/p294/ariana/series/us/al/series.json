[{"y":1990,"t":8,"m":0,"f":8,"mr":null,"fr":423,"mp":0,"fp":0.0003559510567296997},{"y":1991,"t":12,"m":0,"f":12,"mr":null,"fr":298,"mp":0,"fp":0.000543773790103317},{"y":1992,"t":6,"m":0,"f":6,"mr":null,"fr":517,"mp":0,"fp":0.00028063610851262864},{"y":1993,"t":5,"m":0,"f":5,"mr":null,"fr":622,"mp":0,"fp":0.0002342139778902005},{"y":1994,"t":12,"m":0,"f":12,"mr":null,"fr":301,"mp":0,"fp":0.0005714557836087432},{"y":1995,"t":10,"m":0,"f":10,"mr":null,"fr":342,"mp":0,"fp":0.00048199739721405506},{"y":1996,"t":5,"m":0,"f":5,"mr":null,"fr":628,"mp":0,"fp":0.00024311971214626083},{"y":1997,"t":7,"m":0,"f":7,"mr":null,"fr":480,"mp":0,"fp":0.00033757716049382714},{"y":1998,"t":13,"m":0,"f":13,"mr":null,"fr":296,"mp":0,"fp":0.0006236507555768769},{"y":1999,"t":18,"m":0,"f":18,"mr":null,"fr":243,"mp":0,"fp":0.0008533232198729496},{"y":2000,"t":30,"m":0,"f":30,"mr":null,"fr":153,"mp":0,"fp":0.0014203200454502415},{"y":2001,"t":25,"m":0,"f":25,"mr":null,"fr":178,"mp":0,"fp":0.001252191334835963},{"y":2002,"t":22,"m":0,"f":22,"mr":null,"fr":192,"mp":0,"fp":0.0011284366023799754},{"y":2003,"t":31,"m":0,"f":31,"mr":null,"fr":146,"mp":0,"fp":0.0015784114052953156},{"y":2004,"t":34,"m":0,"f":34,"mr":null,"fr":124,"mp":0,"fp":0.001755744900593855},{"y":2005,"t":28,"m":0,"f":28,"mr":null,"fr":151,"mp":0,"fp":0.0014186553174241274},{"y":2006,"t":24,"m":0,"f":24,"mr":null,"fr":187,"mp":0,"fp":0.0011776251226692837},{"y":2007,"t":38,"m":0,"f":38,"mr":null,"fr":127,"mp":0,"fp":0.0017811943376769477},{"y":2008,"t":28,"m":0,"f":28,"mr":null,"fr":173,"mp":0,"fp":0.0013350498259667191},{"y":2009,"t":29,"m":0,"f":29,"mr":null,"fr":151,"mp":0,"fp":0.00145},{"y":2010,"t":35,"m":0,"f":35,"mr":null,"fr":122,"mp":0,"fp":0.0018521458432555432},{"y":2011,"t":37,"m":0,"f":37,"mr":null,"fr":118,"mp":0,"fp":0.0019820012856224555},{"y":2012,"t":31,"m":0,"f":31,"mr":null,"fr":146,"mp":0,"fp":0.0016803989592367736},{"y":2013,"t":46,"m":0,"f":46,"mr":null,"fr":90,"mp":0,"fp":0.002465166130760986},{"y":2014,"t":45,"m":0,"f":45,"mr":null,"fr":95,"mp":0,"fp":0.0023277467411545625},{"y":2015,"t":48,"m":0,"f":48,"mr":null,"fr":95,"mp":0,"fp":0.002443245444365265},{"y":2016,"t":45,"m":0,"f":45,"mr":null,"fr":98,"mp":0,"fp":0.0023259420065126378},{"y":2017,"t":35,"m":0,"f":35,"mr":null,"fr":125,"mp":0,"fp":0.0018278671401712973},{"y":2018,"t":32,"m":0,"f":32,"mr":null,"fr":137,"mp":0,"fp":0.0017132455294999465},{"y":2019,"t":37,"m":0,"f":37,"mr":null,"fr":116,"mp":0,"fp":0.001964532228947648},{"y":2020,"t":35,"m":0,"f":35,"mr":null,"fr":120,"mp":0,"fp":0.001875569369272815},{"y":2021,"t":28,"m":0,"f":28,"mr":null,"fr":156,"mp":0,"fp":0.001471825063078217},{"y":2022,"t":26,"m":0,"f":26,"mr":null,"fr":182,"mp":0,"fp":0.0013616842987325861}]