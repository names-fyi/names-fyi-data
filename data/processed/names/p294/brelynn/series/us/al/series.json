[{"y":2015,"t":7,"m":0,"f":7,"mr":null,"fr":624,"mp":0,"fp":0.00035630662730326784},{"y":2016,"t":6,"m":0,"f":6,"mr":null,"fr":684,"mp":0,"fp":0.0003101256008683517},{"y":2019,"t":5,"m":0,"f":5,"mr":null,"fr":834,"mp":0,"fp":0.0002654773282361686}]