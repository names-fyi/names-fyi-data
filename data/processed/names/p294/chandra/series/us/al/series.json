[{"y":1956,"t":6,"m":0,"f":6,"mr":null,"fr":529,"mp":0,"fp":0.0001636929120969062},{"y":1957,"t":7,"m":0,"f":7,"mr":null,"fr":512,"mp":0,"fp":0.00019092818372746365},{"y":1958,"t":12,"m":0,"f":12,"mr":null,"fr":384,"mp":0,"fp":0.0003334352163160966},{"y":1959,"t":6,"m":0,"f":6,"mr":null,"fr":563,"mp":0,"fp":0.00016953941791466516},{"y":1961,"t":8,"m":0,"f":8,"mr":null,"fr":466,"mp":0,"fp":0.00023299161230195712},{"y":1962,"t":8,"m":0,"f":8,"mr":null,"fr":464,"mp":0,"fp":0.00023982972089816232},{"y":1963,"t":6,"m":0,"f":6,"mr":null,"fr":552,"mp":0,"fp":0.00018711407721574253},{"y":1964,"t":11,"m":0,"f":11,"mr":null,"fr":397,"mp":0,"fp":0.0003420717106695276},{"y":1965,"t":5,"m":0,"f":5,"mr":null,"fr":594,"mp":0,"fp":0.000169485780143046},{"y":1966,"t":5,"m":0,"f":5,"mr":null,"fr":588,"mp":0,"fp":0.00018109380659181456},{"y":1967,"t":14,"m":0,"f":14,"mr":null,"fr":311,"mp":0,"fp":0.0005237364857281808},{"y":1968,"t":21,"m":0,"f":21,"mr":null,"fr":234,"mp":0,"fp":0.0008043511567335682},{"y":1969,"t":15,"m":0,"f":15,"mr":null,"fr":290,"mp":0,"fp":0.0005735479677283677},{"y":1970,"t":29,"m":0,"f":29,"mr":null,"fr":196,"mp":0,"fp":0.0010700317319755},{"y":1971,"t":16,"m":0,"f":16,"mr":null,"fr":278,"mp":0,"fp":0.0006020016555045527},{"y":1972,"t":28,"m":0,"f":28,"mr":null,"fr":185,"mp":0,"fp":0.0011596603851729136},{"y":1973,"t":24,"m":0,"f":24,"mr":null,"fr":208,"mp":0,"fp":0.001045068582625735},{"y":1974,"t":27,"m":0,"f":27,"mr":null,"fr":182,"mp":0,"fp":0.0011979767503771408},{"y":1975,"t":30,"m":0,"f":30,"mr":null,"fr":167,"mp":0,"fp":0.0013728720483250961},{"y":1976,"t":18,"m":0,"f":18,"mr":null,"fr":246,"mp":0,"fp":0.0008369368112707491},{"y":1977,"t":17,"m":0,"f":17,"mr":null,"fr":258,"mp":0,"fp":0.0007390019127108329},{"y":1978,"t":21,"m":0,"f":21,"mr":null,"fr":216,"mp":0,"fp":0.0009655616350177019},{"y":1979,"t":16,"m":0,"f":16,"mr":null,"fr":267,"mp":0,"fp":0.000698263070611853},{"y":1980,"t":18,"m":0,"f":18,"mr":null,"fr":234,"mp":0,"fp":0.0007827448251869891},{"y":1981,"t":6,"m":0,"f":6,"mr":null,"fr":514,"mp":0,"fp":0.00026389866291344125},{"y":1982,"t":8,"m":0,"f":8,"mr":null,"fr":409,"mp":0,"fp":0.00036588154584953124},{"y":1983,"t":7,"m":0,"f":7,"mr":null,"fr":441,"mp":0,"fp":0.0003216320529314464},{"y":1984,"t":11,"m":0,"f":11,"mr":null,"fr":327,"mp":0,"fp":0.0005107252298263534},{"y":1986,"t":14,"m":0,"f":14,"mr":null,"fr":263,"mp":0,"fp":0.000655860582778975},{"y":1987,"t":5,"m":0,"f":5,"mr":null,"fr":562,"mp":0,"fp":0.00023218017181332715},{"y":1988,"t":7,"m":0,"f":7,"mr":null,"fr":460,"mp":0,"fp":0.0003238341968911917},{"y":1989,"t":7,"m":0,"f":7,"mr":null,"fr":461,"mp":0,"fp":0.0003088462386940216},{"y":1995,"t":6,"m":0,"f":6,"mr":null,"fr":534,"mp":0,"fp":0.000289198438328433}]