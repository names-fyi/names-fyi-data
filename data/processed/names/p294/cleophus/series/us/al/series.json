[{"y":1920,"t":6,"m":6,"f":0,"mr":427,"fr":null,"mp":0.0002343658450841764,"fp":0},{"y":1922,"t":7,"m":7,"f":0,"mr":403,"fr":null,"mp":0.00026174095124139994,"fp":0},{"y":1923,"t":8,"m":8,"f":0,"mr":353,"fr":null,"mp":0.00031338138514572234,"fp":0},{"y":1924,"t":10,"m":10,"f":0,"mr":305,"fr":null,"mp":0.00038550501156515033,"fp":0},{"y":1925,"t":6,"m":6,"f":0,"mr":426,"fr":null,"mp":0.00022562328432294213,"fp":0},{"y":1927,"t":6,"m":6,"f":0,"mr":445,"fr":null,"mp":0.00021208158071471493,"fp":0},{"y":1928,"t":6,"m":6,"f":0,"mr":436,"fr":null,"mp":0.00022071806945261918,"fp":0},{"y":1929,"t":5,"m":5,"f":0,"mr":471,"fr":null,"mp":0.00018660894230051504,"fp":0},{"y":1930,"t":5,"m":5,"f":0,"mr":462,"fr":null,"mp":0.00018667164457718873,"fp":0},{"y":1931,"t":5,"m":5,"f":0,"mr":474,"fr":null,"mp":0.00019391870927707106,"fp":0},{"y":1932,"t":8,"m":8,"f":0,"mr":360,"fr":null,"mp":0.0002882363538101243,"fp":0},{"y":1933,"t":9,"m":9,"f":0,"mr":324,"fr":null,"mp":0.0003514526710402999,"fp":0},{"y":1934,"t":11,"m":11,"f":0,"mr":287,"fr":null,"mp":0.00039558384579422445,"fp":0},{"y":1935,"t":6,"m":6,"f":0,"mr":411,"fr":null,"mp":0.00021816595156715876,"fp":0},{"y":1936,"t":6,"m":6,"f":0,"mr":406,"fr":null,"mp":0.00022354694485842027,"fp":0},{"y":1937,"t":6,"m":6,"f":0,"mr":416,"fr":null,"mp":0.00021670820240546104,"fp":0},{"y":1938,"t":9,"m":9,"f":0,"mr":315,"fr":null,"mp":0.00032741559953434226,"fp":0},{"y":1939,"t":9,"m":9,"f":0,"mr":310,"fr":null,"mp":0.0003220035778175313,"fp":0},{"y":1940,"t":8,"m":8,"f":0,"mr":358,"fr":null,"mp":0.0002777392028884877,"fp":0},{"y":1941,"t":8,"m":8,"f":0,"mr":348,"fr":null,"mp":0.0002718129926610492,"fp":0},{"y":1942,"t":11,"m":11,"f":0,"mr":298,"fr":null,"mp":0.0003365458161236041,"fp":0},{"y":1943,"t":9,"m":9,"f":0,"mr":335,"fr":null,"mp":0.00026262036766851475,"fp":0},{"y":1944,"t":7,"m":7,"f":0,"mr":379,"fr":null,"mp":0.00021519259737465033,"fp":0},{"y":1945,"t":13,"m":13,"f":0,"mr":256,"fr":null,"mp":0.0004157605219393629,"fp":0},{"y":1946,"t":11,"m":11,"f":0,"mr":308,"fr":null,"mp":0.0003073913650971077,"fp":0},{"y":1947,"t":12,"m":12,"f":0,"mr":306,"fr":null,"mp":0.0003004657218688968,"fp":0},{"y":1948,"t":7,"m":7,"f":0,"mr":401,"fr":null,"mp":0.00017996709173179762,"fp":0},{"y":1950,"t":13,"m":13,"f":0,"mr":292,"fr":null,"mp":0.00034164673727365904,"fp":0},{"y":1951,"t":9,"m":9,"f":0,"mr":356,"fr":null,"mp":0.0002326543273704891,"fp":0},{"y":1952,"t":7,"m":7,"f":0,"mr":405,"fr":null,"mp":0.0001801755424570796,"fp":0},{"y":1953,"t":7,"m":7,"f":0,"mr":393,"fr":null,"mp":0.00018105633438518441,"fp":0},{"y":1954,"t":11,"m":11,"f":0,"mr":315,"fr":null,"mp":0.00028356362136522996,"fp":0},{"y":1955,"t":8,"m":8,"f":0,"mr":381,"fr":null,"mp":0.00020569782988789467,"fp":0},{"y":1956,"t":5,"m":5,"f":0,"mr":501,"fr":null,"mp":0.00012505314758772478,"fp":0},{"y":1957,"t":7,"m":7,"f":0,"mr":414,"fr":null,"mp":0.00017766497461928934,"fp":0},{"y":1958,"t":7,"m":7,"f":0,"mr":425,"fr":null,"mp":0.00018302567588767454,"fp":0},{"y":1959,"t":10,"m":10,"f":0,"mr":356,"fr":null,"mp":0.0002602269178723847,"fp":0},{"y":1960,"t":13,"m":13,"f":0,"mr":305,"fr":null,"mp":0.0003437789237075235,"fp":0},{"y":1961,"t":8,"m":8,"f":0,"mr":401,"fr":null,"mp":0.00021365238756543105,"fp":0},{"y":1965,"t":6,"m":6,"f":0,"mr":429,"fr":null,"mp":0.00018477457501847746,"fp":0},{"y":1966,"t":9,"m":9,"f":0,"mr":344,"fr":null,"mp":0.00029484029484029485,"fp":0},{"y":1967,"t":6,"m":6,"f":0,"mr":418,"fr":null,"mp":0.00020454777895203354,"fp":0},{"y":1970,"t":5,"m":5,"f":0,"mr":468,"fr":null,"mp":0.00016191185518603672,"fp":0},{"y":1987,"t":5,"m":5,"f":0,"mr":476,"fr":null,"mp":0.00019740998104864183,"fp":0}]