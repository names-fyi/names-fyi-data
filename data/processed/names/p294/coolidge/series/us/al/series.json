[{"y":1923,"t":5,"m":5,"f":0,"mr":495,"fr":null,"mp":0.00019586336571607646,"fp":0},{"y":1924,"t":10,"m":10,"f":0,"mr":306,"fr":null,"mp":0.00038550501156515033,"fp":0},{"y":1925,"t":7,"m":7,"f":0,"mr":388,"fr":null,"mp":0.00026322716504343247,"fp":0},{"y":1926,"t":11,"m":11,"f":0,"mr":291,"fr":null,"mp":0.0003998109984371025,"fp":0}]