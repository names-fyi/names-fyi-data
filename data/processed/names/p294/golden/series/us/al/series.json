[{"y":1919,"t":5,"m":5,"f":0,"mr":516,"fr":null,"mp":0.0001962400408179285,"fp":0},{"y":1924,"t":6,"m":6,"f":0,"mr":437,"fr":null,"mp":0.0002313030069390902,"fp":0}]